components/notifier/src/strategies/notification_strategy.py

```diff
@@ -2,16 +2,25 @@
 
 from datetime import datetime
 
-from models.metric_notification_data import MetricNotificationData
+from shared.model.measurement import Measurement
+from shared.model.metric import Metric
+from shared.model.report import Report
+
+from models.metric_notification_data import MetricNotificationData, NR_OF_MEASUREMENTS_NEEDED_TO_DETERMINE_STATUS_CHANGE
 from models.notification import Notification
 
 
 class NotificationFinder:
     """Handle notification contents and status."""
 
-    def get_notifications(self, reports, measurements, most_recent_measurement_seen: datetime) -> list[Notification]:
+    def get_notifications(
+        self,
+        reports: list[Report],
+        measurements: list[Measurement],
+        most_recent_measurement_seen: datetime,
+    ) -> list[Notification]:
         """Return the reports that have a webhook and metrics that require notifying."""
-        measurements.sort(key=lambda measurement: measurement["end"])
+        measurements.sort(key=lambda measurement: str(measurement["end"]))
```

Comment:
Hmm, a little dirty but if measurement end is always in `datetime.isoformat()` and we always use UTC (from now on), then I guess this is sorted as expected. Perhaps add an inline comment to point that out

Reply:
Done.